docs/dasharo-tools-suite/documentation.md

```diff
@@ -544,6 +544,37 @@ New value is 0x8b.
 SPI Configuration is locked down
 ```
 
+#### Update issues
+
+If you see the following pop-ups during first the boot after the update:
+
+![error-0x03](./images/error-0x03.png)
+![error-0x13](./images/error-0x13.png)
+
+You probably performed an update using a deprecated version of Dasharo Tools
+Suite and have not disabled BIOS lock. Do not worry, nothing bad has happened.
+If you backed up your old firmware, do the following steps:
+
+1. Reboot your device and turn off BIOS lock (you can find this option in
+[Dasharo Security
+Options](../dasharo-menu-docs/dasharo-system-features.md#dasharo-security-options)
+as `BIOS boot medium lock`).
+1. Boot the DTS you backed up your old firmware with and choose option 3, which
```

Comment:
```suggestion
1. Boot the DTS where you backed up your old firmware with and choose option 3, which
```

Reply:
For me it sound strange, but I am definitely not a native speaker

Reply:
The wording is correct, could also say `with which you backed up your old firmware`, but both are valid IMO

Reply:
Yep, strange construction, but I love it.